sql/core/src/test/scala/org/apache/spark/sql/sources/v2/FileDataSourceV2FallBackSuite.scala

```diff
@@ -130,55 +130,9 @@ class FileDataSourceV2FallBackSuite extends QueryTest with SharedSparkSession {
     }
   }
 
-  test("Fall back to v1 when reading file with write only FileDataSourceV2") {
-    val df = spark.range(10).toDF()
-    withTempPath { file =>
-      val path = file.getCanonicalPath
-      // Dummy File writer should fail as expected.
```

Comment:
Now we always forbid file source v2 in `df.write.save`, this test becomes invalid.

Reply:
Shouldn't this test always fall back to v1 as a result of the change? And isn't that fallback what this was testing? I think this test should pass and should still be valid.